r2i/onnxrt/engine.h

```diff
@@ -35,13 +35,27 @@ class Engine : public IEngine {
   r2i::RuntimeError Stop () override;
   std::shared_ptr<r2i::IPrediction> Predict (std::shared_ptr<r2i::IFrame>
       in_frame, r2i::RuntimeError &error) override;
+  r2i::RuntimeError SetLoggingLevel (int logging_level);
+  r2i::RuntimeError SetLogId (const std::string &log_id);
+  r2i::RuntimeError SetIntraNumThreads (int intra_num_threads);
+  r2i::RuntimeError SetGraphOptLevel (int graph_opt_level);
+  int GetLoggingLevel ();
+  int GetIntraNumThreads ();
+  int GetGraphOptLevel ();
+  const std::string GetLogId ();
 
  private:
   enum State {
     STARTED,
     STOPPED
   };
 
+  /* ONNXRT parameters must be initialized in case user does not set any */
+  OrtLoggingLevel logging_level = OrtLoggingLevel::ORT_LOGGING_LEVEL_WARNING;
```

Comment:
Please stick to a standard. In none of the other classes you initialize values here. You do so on the constructor. Try to keep your code consistent.